library/test/src/term/terminfo/searcher/tests.rs

```diff
@@ -13,7 +13,11 @@ fn test_get_dbpath_for_term() {
     }
     assert!(x("screen") == "/usr/share/terminfo/s/screen");
     assert!(get_dbpath_for_term("") == None);
+    // FIXME(skippy) there's no fix for deprecated_safe until tests can be run single threaded
+    #[cfg_attr(not(bootstrap), allow(deprecated_safe))]
```

Comment:
This style of `cfg_attr`s is suspicious - aren't the bootstrap cfgs going to be deleted at the next upgrade?

Instead, use `// SAFETY: FIXME(skippy) This code is incorrect.`